# generated by patch-package 6.4.8 on 2022-03-21 22:23:04
#
# command:
#   npx patch-package fs-extra-plus
#
# declared package:
#   fs-extra-plus: 0.6.0
#
diff --git a/node_modules/fs-extra-plus/dist/index.js b/node_modules/fs-extra-plus/dist/index.js
index e77bf27..7c25d16 100644
--- a/node_modules/fs-extra-plus/dist/index.js
+++ b/node_modules/fs-extra-plus/dist/index.js
@@ -2,6 +2,10 @@ import FastGlob from 'fast-glob';
 import { pathExists, remove } from 'fs-extra';
 import { join as pathJoin, resolve as pathResolve } from 'path';
 import { globCompare } from './util.js';
+
+export * from 'fs-extra';
+export * from 'fs';
+
 /**
  * Simplified and sorted glob function (using fast-glob) for one or more pattern from current directory or a optional cwd one.
  *
diff --git a/node_modules/fs-extra-plus/src/index.ts b/node_modules/fs-extra-plus/src/index.ts
index 7b03c5f..c825de7 100644
--- a/node_modules/fs-extra-plus/src/index.ts
+++ b/node_modules/fs-extra-plus/src/index.ts
@@ -3,8 +3,6 @@ import { pathExists, remove } from 'fs-extra';
 import { join as pathJoin, resolve as pathResolve } from 'path';
 import { globCompare } from './util.js';
 
-
-
 /**
  * Simplified and sorted glob function (using fast-glob) for one or more pattern from current directory or a optional cwd one.
  *